Assets/Scripts/Tutorial System/TutorialUIController.cs

```diff
@@ -125,21 +125,21 @@ private void UpdateWindowPosition(Position _position)
         // This is assuming that the tutorial window's anchor is center screen!
         switch (_position)
         {
+            // NOTE: 1920x1080 can be hard coded because the canvas is set to scale respectively to this size automatically!!!
             case Position.Center:
                 _windowTransfrom.localPosition = new Vector3(0,0, 0);
                 break;
             case Position.Left:
-                Debug.Log("Yas");
-                _windowTransfrom.localPosition = new Vector3(-Screen.width/2 + _windowTransfrom.rect.width/2 + 20, 0, 0);
+                _windowTransfrom.localPosition = new Vector3(-1920/2 + _windowTransfrom.rect.width/2 + 20, 0, 0);
```

Comment:
Is it possible to make this more dynamic instead of giving it a standard value of screen sizes?

Reply:
No, check the comment on start of the switch statement!

Reply:
ahh didn't see apologies